src/idl_gen_fbs.cpp

```diff
@@ -52,16 +52,18 @@ static void GenNameSpace(const Namespace &name_space, std::string *_schema,
 }
 
 // Generate a flatbuffer schema from the Parser's internal representation.
-std::string GenerateFBS(const Parser &parser, const std::string &file_name) {
-  // Proto namespaces may clash with table names, so we have to prefix all:
-  for (auto it = parser.namespaces_.begin(); it != parser.namespaces_.end();
-       ++it) {
-    for (auto comp = (*it)->components.begin(); comp != (*it)->components.end();
-         ++comp) {
-      (*comp) = "_" + (*comp);
-    }
-  }
+std::string GenerateFBS(const Parser &parser, const std::string &file_name, const bool &escape_proto_identifiers) {
+ // Proto namespaces may clash with table names, so we have to prefix all:
```

Comment:
please fix your editor settings, it seems a lot of lines have changed indentation but no actual code changes. indent == 2 spaces, no trailing spaces.